playground/nuxt.config.ts

```diff
@@ -2,7 +2,8 @@ export default defineNuxtConfig({
   modules: ['@nuxtjs/turnstile'],
   runtimeConfig: {
     turnstile: {
-      secretKey: '1x0000000000000000000000000000000AA'
+      secretKey: '1x0000000000000000000000000000000AA',
+      // secretKeyPath: 'path/to/secret/key' // might be: 'src/development/secrets/turnstile-secret-key.secret' to use the template
```

Comment:
```suggestion
      // secretKeyPath: 'path/to/secret/key'
```
https://github.com/dargmuesli/dargstack_template is a devops template by me, not related to nuxt :wink:

Reply:
Well you are right. It just seemed like a good idea to me. I'll remove that comment and the corresponding template file 😁

Reply:
This thread can be resolved :+1: